packages/cli/src/zosfiles/create/ds/ds.definition.ts

```diff
@@ -0,0 +1,65 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { ICommandDefinition } from "@zowe/imperative";
+import { CreateDefaults } from "@zowe/zos-files-for-zowe-sdk";
+import { ZosFilesCreateExtraOptions, ZosFilesCreateOptions } from "../Create.options";
+import i18nTypings from "../../-strings-/en";
+
+// Does not use the import in anticipation of some internationalization work to be done later.
+const strings = (require("../../-strings-/en").default as typeof i18nTypings).CREATE;
+
+/**
+ * Create dataSet command definition containing its description, examples and/or options
+ * @type {ICommandDefinition}
+ */
+export const DsDefinition: ICommandDefinition = {
+    name: "data-set",
+    aliases: ["ds"],
+    description: strings.ACTIONS.DATA_SET_LIKE.DESCRIPTION,
+    type: "command",
+    handler: __dirname + "/ds.handler",
+    profile: {
+        optional: ["zosmf"]
+    },
+    positionals: [
+        {
+            name: "dataSetName",
+            type: "string",
+            description: strings.POSITIONALS.DATASETNAME,
+            required: true
+        }
+    ],
+    options: [
+        ZosFilesCreateExtraOptions.size,
+        ZosFilesCreateOptions.volser,
+        {...ZosFilesCreateOptions.primary, defaultValue: CreateDefaults.DATA_SET.C.primary},
+        ZosFilesCreateOptions.secondary,
+        {...ZosFilesCreateOptions.dirblk, defaultValue: CreateDefaults.DATA_SET.C.dirblk},
+        {...ZosFilesCreateOptions.recfm, defaultValue: CreateDefaults.DATA_SET.C.recfm},
+        {...ZosFilesCreateOptions.blksize, defaultValue: CreateDefaults.DATA_SET.C.blksize},
+        {...ZosFilesCreateOptions.lrecl, defaultValue: CreateDefaults.DATA_SET.C.lrecl},
```

Comment:
Please remove the default values for these options - the "create ds" command shouldn't have any as mentioned in #771:
> Now, for this zowe files create data-set command, I'm thinking all of the options should be left without defaults. The expectation being that users will either leverage the --like option and specify optional overrides or not leverage the --like option and fully specify the attributes. If they didn't want to fully specify the attributes they could always opt to use one of the template type creation commands.